ddtrace/contrib/django/patch.py

```diff
@@ -477,9 +477,34 @@ def traced_template_render(django, pin, wrapped, instance, args, kwargs):
 
 
 def instrument_view(django, view):
+    """
+    Helper to wrap Django views.
+
+    It is *very* important that we wrap all views in the MRO for the passed in view.
+    """
+    # DEV: Reverse the MRO so we always wrap the base most class first
+    #      __mro__ = (<class 'django.views.generic.base.TemplateView'>,
+    #                 <class 'django.views.generic.base.TemplateResponseMixin'>,
+    #                 <class 'django.views.generic.base.ContextMixin'>,
+    #                 <class 'django.views.generic.base.View'>,
+    #                 <class 'object'>)
+    #   If we wrapped `django.views.generic.base.TemplateView` first, then we would
+    #   wrapped `django.views.generic.base.View.dispatch` and assign as a property to
+    #   `django.views.generic.base.TemplateView` causing any in-between views `dispatch`
+    #   method from being called
+    if isfunction(view):
+        return _instrument_view(django, view)
+
+    for cls in reversed(getmro(view)):
+        _instrument_view(django, cls)
+
+    return view
```

Comment:
```suggestion
    for cls in reversed(getmro(view))[:-1]:
        _instrument_view(django, cls)

    return _instrument_view(django, view)
```

Reply:
This feels like a gross implementation, but basically, go through the MRO, but skip the current view class, and then instrument and return the view.

This is so we can return the `wrapt.FunctionWrapper` from `_instrument_view` L549